### PR TITLE
Task with servlets and db

### PR DESCRIPTION
**Pull request by daria-naumenko**

### CODE REVIEW

File: src/main/java/secondTask/CounterServlet.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    }
+
+    public void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException {
+        resp.setContentType("text/html; charset=UTF-8");
+        PrintWriter writer = resp.getWriter();
+        writer.println("Посетитель сайта номер: " + (++сount));
+    }
+
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        doGet(request, response);
```
Comment (DHrytsuk) [inline src/main/java/secondTask/CounterServlet.java]: Try to avoid this.
In general, no need to override doPost method, if we don't use it

**Comment by DHrytsuk** [comment]
Don't see neither jsp no html files. 
Are there any of them?

**Comment by daria-naumenko** [comment]
> Don't see neither jsp no html files. Are there any of them?

in this task i dont use jsp file

**Comment by DHrytsuk** [comment]
> in this task i dont use jsp file

But how do you proceed with POST request in this way?)

**Comment by daria-naumenko** [comment]
Why when I type in the url address /WEB-INF/student.jsp I get nothing "Error 404"

**Comment by DHrytsuk** [comment]
> Why when I type in the url address /WEB-INF/student.jsp I get nothing "Error 404"

Because everything taht we place under WEB-INF (I mean jsp, html,css files) is secured for the application and could not be accessagle via direct link. We need to access it via Servlet (call doGet method, which will open our page)

**Pull request merged**